build_projects/dotnet-host-build/TestTargets.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.DotNet.Cli.Build.Framework;
+
+using static Microsoft.DotNet.Cli.Build.FS;
+using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+using static Microsoft.DotNet.Cli.Build.Utils;
+using Microsoft.DotNet.Cli.Build;
+
+namespace Microsoft.DotNet.Host.Build
+{
+    public class TestTargets
+    {
+        public static readonly string[] TestProjects = new[]
+        {
+            "HostActivationTests"
+        };
+
+        [Target(
+            nameof(PrepareTargets.Init),
+            nameof(RestoreTestAssets),
+            nameof(RestoreTests),
+            nameof(BuildTests),
+            nameof(RunTests))]
+        public static BuildTargetResult Test(BuildTargetContext c) => c.Success();
+
+        [Target]
+        public static BuildTargetResult RestoreTestAssets(BuildTargetContext c)
+        {
+            var dotnet = DotNetCli.Stage0;
+            CleanBinObj(c, Path.Combine(Dirs.RepoRoot, "TestAssets"));
+
+            dotnet.Restore(
+                    "--fallbacksource", Dirs.CorehostLocalPackages,
+                    "--fallbacksource", Dirs.CorehostDummyPackages)
+                .WorkingDirectory(Path.Combine(Dirs.RepoRoot, "TestAssets"))
+                .Execute()
+                .EnsureSuccessful();
+
+            return c.Success();
+        }
+
+        [Target]
+        public static BuildTargetResult RestoreTests(BuildTargetContext c)
+        {
+            var dotnet = DotNetCli.Stage0;
+            CleanBinObj(c, Path.Combine(Dirs.RepoRoot, "test"));
+
+            dotnet.Restore(
+                    "--fallbacksource", Dirs.CorehostLocalPackages,
```

Comment:
I thought the tests don't use the local corehost/Microsoft.NETCore.App.  It shouldn't need this right?